build.sbt

```diff
@@ -0,0 +1,57 @@
+import com.softwaremill.SbtSoftwareMillCommon.commonSmlBuildSettings
+
+val tapirVersion = "1.1.1"
+
+lazy val commonSettings = commonSmlBuildSettings ++ Seq(
+  organization := "com.softwaremill.mqperf",
+  scalaVersion := "2.13.8"
+)
+
+val scalaTest = "org.scalatest" %% "scalatest" % "3.2.13" % Test
+
+//
+
+lazy val dockerSettings = Seq(
+  dockerExposedPorts := Seq(8080),
+  dockerBaseImage := "eclipse-temurin:11.0.16.1_1-jre-jammy",
+  dockerUsername := Some("mqperf"),
+  dockerRepository := Some("europe-west2-docker.pkg.dev/sml-gke-iac"),
+  dockerUpdateLatest := true,
+  Docker / version := git.gitHeadCommit.value.map(head => head.take(8) + "-" + (System.currentTimeMillis() / 1000)).getOrElse("latest")
```

Comment:
I'm not sure if this dockerSettings will create correct image name, in the it end should look something like this: `europe-west2-docker.pkg.dev/sml-gke-iac/mqperf/kafka:d5aebf3-1662635645`. Also if this is a public project WDYT about pushing it to public Docker Hub instead of SML's GCP artifact registry?

Reply:
Good point as well, done: https://hub.docker.com/repository/docker/softwaremill/mqperf-kafka